src/components/pages/UserProfile/PersonalInformationPanel.js

```diff
@@ -52,7 +52,7 @@ export default class PersonalInformationPanel extends PureComponent {
                   <PTButton className="btn btn-danger" onClick={() => onCancel(name)}>
                     <i className="fa fa-ban" /> Cancel
                   </PTButton>
-                  <PTButton className="btn btn-success">
+                  <PTButton className="btn btn-success" onClick={() => onClick()}>
```

Comment:
I don't think it's a good solution cause we loose "event" argument. Prefer onClick={onClick}